.github/workflows/lint-test.yaml

```diff
@@ -46,5 +46,6 @@ jobs:
           helm repo add kedacore https://kedacore.github.io/charts
           helm repo update
           helm install keda kedacore/keda
+          kubectl apply -f ci/pv.yaml
```

Comment:
Meant to add a comment here, but it might be nice to note we're doing this since the PVC tests assumes one already exists.

Not blocking though.